src/firebolt_db/firebolt_dialect.py

```diff
@@ -96,8 +94,8 @@ def __init__(
         self.context: Union[ExecutionContext, Dict] = context or {}
 
     @classmethod
-    def dbapi(cls) -> ModuleType:
-        return firebolt_db
+    def dbapi(cls) -> Any:
```

Comment:
Isn't it a `ModuleType`?

Reply:
Problem with this is once we wrap it in a wrapper in async_dialect it's not module type anymore, but that's an intended behaviour of sqlalchemy. Do you think doing something like `ModuleType | AsyncAPIWrapper` is clearer?

Reply:
Can `AsyncApiWrapper` inherit `ModuleType`?
That's not a big issue here, I just don't like `Any`. Maybe we can just say `-> AsyncAPIWrapper`?

Reply:
Oh, I see we can't.
Then yeah, composing two like `ModuleType | AsyncAPIWrapper` can also be an option. Yet i think that having a single return type is better

Reply:
Yeah, inheriting `ModuleType` seems to have done the trick. I agree, `Any` should be avoided where possible.